components/menu/menu-item-radio-mixin.js

```diff
@@ -0,0 +1,27 @@
+import { MenuItemSelectableMixin } from './menu-item-selectable-mixin.js';
+
+export const MenuItemRadioMixin = superclass => class extends MenuItemSelectableMixin(superclass) {
+
+	constructor() {
+		super();
+		this.role = 'menuitemradio';
+	}
+
+	firstUpdated() {
+		super.firstUpdated();
+		this.addEventListener('d2l-menu-item-change', this._onChange);
+		this.addEventListener('d2l-menu-item-select', this._onSelectRadio);
+	}
+
+	_onSelectRadio(e) {
+		this.selected = true;
+		this.__onSelect(e);
+	}
+
+	_onChange(e) {
+		const items = this.parentNode.querySelectorAll('[role="menuitemradio"]');
```

Comment:
Just to confirm: does this only run for the item that changed? Just want to make sure it isn't running for each item each time selection changes.

Reply:
That's correct, it only runs for the item that changed.

Reply:
which makes me think this could move to `_onSelectRadio` and not have to deal with event listeners